test/common/TestBed.cpp

```diff
@@ -220,7 +220,7 @@ namespace RcclUnitTesting
     }
   }
 
-  void TestBed::ExecuteCollectives()
+  void TestBed::ExecuteCollectives(std::vector<int> const &currentRanks)
   {
     int const cmd = TestBedChild::CHILD_EXECUTE_COLL;
     ++TestBed::NumTestsRun();
```

Comment:
We will need to change this to count only times when the "first" rank is selected for ExecuteCollectives, otherwise we duplicate when rank is specified.